.chloggen/kube-distro.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
This should be a breaking change. We discussed that the k8s distro should be enabled for the first time with v1beta1 release

Reply:
not a breaking change :) we're going from core to k8s distro